pkg/virt-handler/selinux/labels.go

```diff
@@ -171,15 +171,16 @@ type SELinux interface {
 	IsPermissive() bool
 }
 
-func RelabelFiles(newLabel string, continueOnError bool, files ...*safepath.Path) error {
-	relabelArgs := []string{"selinux", "relabel", newLabel}
+func RelabelFiles(continueOnError bool, files ...*safepath.Path) error {
```

Comment:
I would rename this function to something less generic since it now is specialized for `unprivilegedContainerSELinuxLabel`

Reply:
Do you have a preferred name? `RelabelFilesUnpreviliged`?

Personally, I prefer to keep the `RelabelFiles`, because the callers of this function do not care if which label is used, they just want to the files to be accessible.

Reply:
Actually, I don't have a strong opinion. 
/lgtm

Reply:
> Do you have a preferred name? `RelabelFilesUnpreviliged`?
> 
> Personally, I prefer to keep the `RelabelFiles`, because the callers of this function do not care if which label is used, they just want to the files to be accessible.

You're right that the caller doesn't care which label is used.
However, the caller might wonder why this operation is necessary at all. By calling it something like `RelabelFilesUnpreviliged` the caller doesn't know (and doesn't care) about the specific label being used, but is aware of why the operation is needed in the first place. In addition, it's a public function, which means it could be used by other packages in the future, therefore I think it's better to provide this context.

Does this make sense? WDYT?

Reply:
Thanks for your patience!